README.md

```diff
@@ -124,6 +124,8 @@ In addition, it is possible to specify this optional input:
 -   **SHELL**: The shell type to use. By default, it is **bash**.
 -   **PROJECT**: The project to use when running npm scripts. If set, the executed npm script will be `{PROJECT}:{SCRIPT_NAME}` instead of `{SCRIPT_NAME}`.
 -   **CHECKOUT_REF**: The ref of the branch/tag to check out before running the build. See the ref parameter of the [checkout action](https://github.com/actions/checkout). By default, it is `''`.
+-   **RUN_ON**: the _labels_ to select the correct _github-runner_ that will execute this workflow. Default is **zupit-agents**.
+-   **RUNNERS_CONTAINER_GROUP**: the _labels_ to select the correct _github-runner group_  that will execute this workflow. Default is **Container**.
```

Comment:
Qui scriverei "The runners group used to execute this workflow. Default is **Container**"